lib/poker_hands/hand_rankings/royal_flush.rb

```diff
@@ -5,9 +5,10 @@ def check(hand)
     
     return false if hand.any? { |card| card.suit != hand.first.suit}
 
-    hand.size.times do |card|
-      ranks_in_hand << hand[card].rank
-    end
+    ranks_in_hand = hand.map(&:rank)
+    #hand.each do |card|
+    #  ranks_in_hand << card.rank
+    #end
 
     (royal_flush_ranks - ranks_in_hand).empty? ? true : false
```

Comment:
We don't need to explicitly return the boolean here. The return value of a method in Ruby is that of the last evaluated expression.